src/metaswitch/clearwater/etcd_shared/plugin_utils.py

```diff
@@ -29,8 +30,7 @@ def run_command(command, namespace=None, log_error=True):
 
     # Pass the close_fds argument to avoid the pidfile lock being held by
     # child processes
-    p = subprocess.Popen(command,
-                         shell=True,
+    p = subprocess.Popen(shlex.split(command),
```

Comment:
The comment on this function is incorrect now (the command isn't run in a shell, nor will shell replacements be made).  I'd still like to see this function take the command as an array and not have any reference to `shlex` in it.